<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>boost.git - boost</title>
<meta name='generator' content='cgit v0.10.2'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit-data/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://pkgs.fedoraproject.org/cgit/boost.git/atom/boost-1.48.0-xtime.patch?h=master' type='application/atom+xml'/>
</head>
<body>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/'><img src='/cgit-data/cgit.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/'>index</a> : <a title='boost.git' href='/cgit/boost.git/'>boost.git</a></td><td class='form'><form method='get' action=''>
<input type='hidden' name='id' value='f68c4dd92574829a34324290c15e360d0fedf643'/><select name='h' onchange='this.form.submit();'>
<option value='boost-1_34_0-exp'>boost-1_34_0-exp</option>
<option value='f10'>f10</option>
<option value='f11'>f11</option>
<option value='f12'>f12</option>
<option value='f13'>f13</option>
<option value='f14'>f14</option>
<option value='f15'>f15</option>
<option value='f16'>f16</option>
<option value='f17'>f17</option>
<option value='f18'>f18</option>
<option value='f19'>f19</option>
<option value='f20'>f20</option>
<option value='f21'>f21</option>
<option value='f22'>f22</option>
<option value='f7'>f7</option>
<option value='f8'>f8</option>
<option value='f9'>f9</option>
<option value='master' selected='selected'>master</option>
<option value='pmachata/boost153'>pmachata/boost153</option>
<option value='pmachata/master-next'>pmachata/master-next</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>boost</td><td class='sub right'>Jesse Keating</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/boost.git/'>summary</a><a href='/cgit/boost.git/refs/?id=f68c4dd92574829a34324290c15e360d0fedf643'>refs</a><a href='/cgit/boost.git/log/boost-1.48.0-xtime.patch'>log</a><a class='active' href='/cgit/boost.git/tree/boost-1.48.0-xtime.patch?id=f68c4dd92574829a34324290c15e360d0fedf643'>tree</a><a href='/cgit/boost.git/commit/boost-1.48.0-xtime.patch?id=f68c4dd92574829a34324290c15e360d0fedf643'>commit</a><a href='/cgit/boost.git/diff/boost-1.48.0-xtime.patch?id=f68c4dd92574829a34324290c15e360d0fedf643'>diff</a><a href='/cgit/boost.git/stats/boost-1.48.0-xtime.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/boost.git/log/boost-1.48.0-xtime.patch'>
<input type='hidden' name='id' value='f68c4dd92574829a34324290c15e360d0fedf643'/><select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/boost.git/tree/?id=f68c4dd92574829a34324290c15e360d0fedf643'>root</a>/<a href='/cgit/boost.git/tree/boost-1.48.0-xtime.patch?id=f68c4dd92574829a34324290c15e360d0fedf643'>boost-1.48.0-xtime.patch</a></div><div class='content'>blob: 01ad47fb90a8b4492af002f40b6e731c6abe7e70 (<a href='/cgit/boost.git/plain/boost-1.48.0-xtime.patch?id=f68c4dd92574829a34324290c15e360d0fedf643'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
<a id='n16' href='#n16'>16</a>
<a id='n17' href='#n17'>17</a>
<a id='n18' href='#n18'>18</a>
<a id='n19' href='#n19'>19</a>
<a id='n20' href='#n20'>20</a>
<a id='n21' href='#n21'>21</a>
<a id='n22' href='#n22'>22</a>
<a id='n23' href='#n23'>23</a>
<a id='n24' href='#n24'>24</a>
<a id='n25' href='#n25'>25</a>
<a id='n26' href='#n26'>26</a>
<a id='n27' href='#n27'>27</a>
<a id='n28' href='#n28'>28</a>
<a id='n29' href='#n29'>29</a>
<a id='n30' href='#n30'>30</a>
<a id='n31' href='#n31'>31</a>
<a id='n32' href='#n32'>32</a>
<a id='n33' href='#n33'>33</a>
<a id='n34' href='#n34'>34</a>
<a id='n35' href='#n35'>35</a>
<a id='n36' href='#n36'>36</a>
<a id='n37' href='#n37'>37</a>
<a id='n38' href='#n38'>38</a>
<a id='n39' href='#n39'>39</a>
<a id='n40' href='#n40'>40</a>
<a id='n41' href='#n41'>41</a>
<a id='n42' href='#n42'>42</a>
<a id='n43' href='#n43'>43</a>
<a id='n44' href='#n44'>44</a>
<a id='n45' href='#n45'>45</a>
<a id='n46' href='#n46'>46</a>
<a id='n47' href='#n47'>47</a>
<a id='n48' href='#n48'>48</a>
<a id='n49' href='#n49'>49</a>
<a id='n50' href='#n50'>50</a>
<a id='n51' href='#n51'>51</a>
<a id='n52' href='#n52'>52</a>
<a id='n53' href='#n53'>53</a>
<a id='n54' href='#n54'>54</a>
<a id='n55' href='#n55'>55</a>
<a id='n56' href='#n56'>56</a>
<a id='n57' href='#n57'>57</a>
<a id='n58' href='#n58'>58</a>
<a id='n59' href='#n59'>59</a>
<a id='n60' href='#n60'>60</a>
<a id='n61' href='#n61'>61</a>
<a id='n62' href='#n62'>62</a>
<a id='n63' href='#n63'>63</a>
<a id='n64' href='#n64'>64</a>
<a id='n65' href='#n65'>65</a>
<a id='n66' href='#n66'>66</a>
<a id='n67' href='#n67'>67</a>
<a id='n68' href='#n68'>68</a>
<a id='n69' href='#n69'>69</a>
</pre></td>
<td class='lines'><pre><code>diff -up boost_1_48_0/boost/thread/xtime.hpp\~ boost_1_48_0/boost/thread/xtime.hpp
--- boost_1_48_0/boost/thread/xtime.hpp~	2008-06-18 15:01:08.000000000 +0200
+++ boost_1_48_0/boost/thread/xtime.hpp	2012-05-24 20:45:18.000000000 +0200
@@ -18,18 +18,6 @@
 
 namespace boost {
 
-enum xtime_clock_types
-{
-    TIME_UTC=1
-//    TIME_TAI,
-//    TIME_MONOTONIC,
-//    TIME_PROCESS,
-//    TIME_THREAD,
-//    TIME_LOCAL,
-//    TIME_SYNC,
-//    TIME_RESOLUTION
-};
-
 struct xtime
 {
 #if defined(BOOST_NO_INT64_T)
diff -up boost_1_48_0/libs/thread/src/pthread/timeconv.inl\~ boost_1_48_0/libs/thread/src/pthread/timeconv.inl
--- boost_1_48_0/libs/thread/src/pthread/timeconv.inl~	2010-04-01 17:04:15.000000000 +0200
+++ boost_1_48_0/libs/thread/src/pthread/timeconv.inl	2012-05-24 20:55:43.000000000 +0200
@@ -20,8 +20,8 @@ const int NANOSECONDS_PER_MICROSECOND =
 inline void to_time(int milliseconds, boost::xtime&amp; xt)
 {
     int res = 0;
-    res = boost::xtime_get(&amp;xt, boost::TIME_UTC);
-    BOOST_ASSERT(res == boost::TIME_UTC);
+    res = boost::xtime_get(&amp;xt, TIME_UTC);
+    BOOST_ASSERT(res == TIME_UTC);
 
     xt.sec += (milliseconds / MILLISECONDS_PER_SECOND);
     xt.nsec += ((milliseconds % MILLISECONDS_PER_SECOND) *
@@ -57,8 +57,8 @@ inline void to_timespec_duration(const b
 {
     boost::xtime cur;
     int res = 0;
-    res = boost::xtime_get(&amp;cur, boost::TIME_UTC);
-    BOOST_ASSERT(res == boost::TIME_UTC);
+    res = boost::xtime_get(&amp;cur, TIME_UTC);
+    BOOST_ASSERT(res == TIME_UTC);
 
     if (boost::xtime_cmp(xt, cur) &lt;= 0)
     {
@@ -88,8 +88,8 @@ inline void to_duration(boost::xtime xt,
 {
     boost::xtime cur;
     int res = 0;
-    res = boost::xtime_get(&amp;cur, boost::TIME_UTC);
-    BOOST_ASSERT(res == boost::TIME_UTC);
+    res = boost::xtime_get(&amp;cur, TIME_UTC);
+    BOOST_ASSERT(res == TIME_UTC);
 
     if (boost::xtime_cmp(xt, cur) &lt;= 0)
         milliseconds = 0;
@@ -110,8 +110,8 @@ inline void to_microduration(boost::xtim
 {
     boost::xtime cur;
     int res = 0;
-    res = boost::xtime_get(&amp;cur, boost::TIME_UTC);
-    BOOST_ASSERT(res == boost::TIME_UTC);
+    res = boost::xtime_get(&amp;cur, TIME_UTC);
+    BOOST_ASSERT(res == TIME_UTC);
 
     if (boost::xtime_cmp(xt, cur) &lt;= 0)
         microseconds = 0;
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated  by cgit v0.10.2 at 2015-04-11 18:22:54 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
